tests/static-check/run_checks.sh

```diff
@@ -8,7 +8,7 @@ function check_with_gcc() {
   rm -f config.cache
   make clean
   ./configure -C --enable-debug CC=gcc
-  local gcc_exceptions="-Wno-sign-compare -Wno-implicit-fallthrough -Wno-cast-function-type -Wno-type-limits"
```

Comment:
Only the `implicit-fallthrough` warning should be removed here.

Reply:
There is a conflict here now, since we merged: https://github.com/cfengine/core/pull/4601